### PR TITLE
Use post_name with the 'r'.

### PR DESCRIPTION
**Pull request by octalmage**
I had to use the r35252 format for the slug.

### CODE REVIEW

File: src/class-shortcode.php [hunk 1]
```diff
@@ -99,7 +99,7 @@ public static function render_results( $atts ) {
 					$query_args = array(
 						'posts_per_page'   => 10,
 						'post_type'        => 'result',
-						'post_parent'      => $rev_id,
+						'post_parent'      => $revision->post_name,
```
Comment (danielbachhuber) [inline src/class-shortcode.php]: The pull request says:

> Use post_name with the 'r'

But the commit is changing the `post_parent` argument?

`post_parent` needs to be/remain numeric.

**Comment by octalmage** [comment]
Ah my bad. I see what happened here.

screen shot 2017-08-02 at 9 26 56 am

So I ended up actually making the revision post the parent of the results, instead of setting the post parent to the revision. This gives us a nice neat display in the backend too.

**Comment by danielbachhuber** [comment]
> So I ended up actually making the revision post the parent of the results, instead of setting the post parent to the revision. This gives us a nice neat display in the backend too.

Oh, ok. That works too I suppose.

**Comment by octalmage** [comment]
Woohoo, looking good! https://wpunittestapi.wpengine.com/

**Pull request merged**